html/biography.html

```diff
@@ -4,9 +4,23 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Document</title>
-    <link rel="stylesheet" href="main.css">
+    <link rel="stylesheet" href="/css/main.css">
 </head>
```

Comment:
Hey I think you css link should be ../css/main.css since it is up one folder.  When you run your app you should have a aqua background.

Reply:
I will take a look.  As of yesterday when I was adding it wasn't "auto" populating.  When I just did the / it would start to auto-populate.  I haven't even run any code yet but I will check it prior to the merge

Reply:
Are you putting two dots?  I was getting the "auto" populate when I did that.